.env_example

```diff
@@ -8,12 +14,16 @@ TESTNET_PRIVATE_KEY=
 TESTNET2_ACCOUNT_ADDRESS=
 TESTNET2_PRIVATE_KEY=
 
-DEVNET_ACCOUNT_ADDRESS=0x7e00d496e324876bbc8531f2d9a82bf154d1a04a50218ee74cdd372f75a551a
-DEVNET_PRIVATE_KEY=0xe3e70682c2094cac629f6fbed82c07cd
+DEVNET_ACCOUNT_ADDRESS=
+DEVNET_PRIVATE_KEY=
 
 # These will be used if above variables are not set for the given network
 ACCOUNT_ADDRESS=
 PRIVATE_KEY=
 
-# A default evm address to deploy an eoa for
+# A default evm address to deploy an eoa for with corresponding private key
 EVM_ADDRESS=
+EVM_PRIVATE_KEY=
```

Comment:
if we add the `PRIVATE_KEY` we may want to remove the `ADDRESS` as the address can be derived from the PRIVATE_KEY